packages/actor-query-operation-group/lib/GroupsState.ts

```diff
@@ -86,11 +87,13 @@ export class GroupsState {
         await Promise.all(this.pattern.aggregates.map(async(aggregate) => {
           // If distinct, check first whether we have inserted these values already
           if (aggregate.distinct) {
-            const hash = this.hashBindings(bindings);
-            if (this.distinctHashes!.get(groupHash)!.has(hash)) {
+            const bindingsHash = this.hashBindings(bindings);
+            const hashBindingsAndAggregate = createHash('sha256')
```

Comment:
We should avoid directly hashing in this actor.
All hashing should be delegated to `bus-hash-bindings`, so downstream users can configure different hashing methods.